empirical-results.tex

```diff
@@ -1 +1,29 @@
-\section{Empirical Results}
\ No newline at end of file
+\section{Empirical Results}
+
+\newcommand{\rref}[1]{}
```

Comment:
?

Reply:
Some refs are included depend on another branch. I used this macro to be able to easily disable them until the other branch is merged

Reply:
Instead of creating a dependency or rebasing on top of the other